app/static/js/events/scheduler.js

```diff
@@ -550,10 +550,32 @@ function addInfoBox($sessionElement, session) {
         html: true,
         title: session.title
     });
-    var speakers = _.map(session.speakers, 'name');
     var content = "";
-    if(speakers.length > 0) {
-        content += "By " + _.join(speakers, ', ') + "<br><br>";
+    if(!_.isNull(session.short_abstract)) {
```

Comment:
Implement this just like speakers ... If there is only long_abstract and no short_abstract, show a truncated version of it.